HunterPie.Integrations/Datasources/MonsterHunterWorld/Entity/Player/MHWPlayer.cs

```diff
@@ -166,6 +166,12 @@ protected set
         }
     }
 
+    public override CombatStatus CombatStatus
+    {
+        get => 0;
```

Comment:
```suggestion
        get => CombatStatus.None;
```
Explicit value is better than implicit